config/flags.go

```diff
@@ -175,7 +175,6 @@ func addNodeFlags(fs *pflag.FlagSet) {
 	fs.Duration(BenchlistMinFailingDurationKey, constants.DefaultBenchlistMinFailingDuration, "Minimum amount of time messages to a peer must be failing before the peer is benched")
 
 	// Router
-	// TODO: Remove this flag in the future
```

Comment:
Why's this TODO removed? Do we no longer intend to remove this flag?

Reply:
The flag was already removed (but the todo wasn't)